windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -976,8 +978,6 @@ procedure TMergeKeyboardInfo.AddPlatformSupport;
       AddNewPair('windows', 'full');
       AddNewPair('macos', 'full');
       AddNewPair('linux', 'full');
-      AddNewPair('desktopWeb', 'full');
```

Comment:
Should the web checks be removed in the block at line 957?
```
      // Validate web/mobile targets. No kmx implies web/mobile targets
      if ((numberKMXFiles = 0) or (p.GetValue('web') <> nil) or
          (p.GetValue('iphone') <> nil) or (p.GetValue('ipad') <> nil) or
          (p.GetValue('androidphone') <> nil) or (p.GetValue('androidtablet') <> nil) or
          (p.GetValue('mobile') <> nil) or (p.GetValue('tablet') <> nil)) and
          (numberJSFiles = 0) then
        raise EInvalidKeyboardInfo.Create(
          'Web/mobile targets should have at least 1 .js keyboard file in the package');
```

Reply:
Good pick. Probably!

Reply:
This is a little tricky. Currently the test checks for .js outside the package as well as inside. I think I'd like to leave changes to this for a rainy day -- it is working as is without breaking the build.